packages/airnode-deployer/src/handlers/aws/index.ts

```diff
@@ -63,7 +56,7 @@ async function initializeProvider(payload: InitializeProviderPayload) {
   const stateWithConfig = { ...state, config: parsedConfig };
 
   const goInitializedState = await go(() => handlers.initializeProvider(stateWithConfig), {
-    delay: { type: 'static', delayMs: DEFAULT_RETRY_DELAY_MS },
+    delay: { type: 'static', delayMs: 50 },
```

Comment:
Why haven't you used a constant for these?

Reply:
It feels these could be different values for each handler, but they should be the same for AWS and GCP. I'll make constants for these.

Reply:
Mhh, I just realized there are no retries specified for these functions, so the delay doesn't make sense... It feels to me we should not even retry those. WDYT?

Reply:
Handlers? I agree.